### PR TITLE
Add multilanguage support

### PR DESCRIPTION
**Pull request by benhid**

### CODE REVIEW

**Comment by vincentdoerig** [comment]
Thank you for the PR! Let me quickly set up eslint so that we all use the same code style. 

I see you divided the sections using `<section>`, so we will probably need to add another lobotomized owl selector. The elements inside the `<section>` aren't properly spaced otherwise.

image
Left: your version, Right: current

**Comment by benhid** [comment]
Sorry, that was definitely not intended! I accidentally took some commits from another pull request I made (https://github.com/vincentdoerig/latex-css/pull/11). 

Commits have been reverted to keep PRs separate.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,20 @@ yarn add latex.css
 
 Add any optional classnames to elements with special styles (author subtitle, abstract, lemmas, theorems, etc.). A list of supported class-based elements can be found [here](https://latex.now.sh/#class-based-elements).
 
+## Languages
+
+Theorems, Definitions, Lemmas and Proofs labels can be change to other [supported languages](lang) adding the following code in the head of your project.
```
Comment (vincentdoerig) [inline README.md:42]: Better: `The labels of theorems, definitions, lemmas and proofs can be changed to other [supported languages](lang) by including the following snippet in addition to the main CSS file.`

File: lang/es.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
 :lang(es) .theorem:before { content: 'Teorema ' counter(theorem) '. ' !important; }
```
Comment (vincentdoerig) [inline lang/es.css:1]: use `::before` instead of `:before`

**Pull request merged**